.github/workflows/operate-playwright.yml

```diff
@@ -2,11 +2,21 @@
 name: Operate Visual regression tests
 on:
   push:
-    paths-ignore:
-      - '.github/workflows/zeebe-*'
-      - 'dist/**'
-      - 'zeebe/**'
-      - 'spring-boot-starter-camunda-sdk/**'
+    branches:
+      - main
+    paths:
+      - "operate/client/**"
+      - "!.github/workflows/zeebe-*"
+      - "!dist/**"
+      - "!zeebe/**"
+      - "!spring-boot-starter-camunda-sdk/**"
+  pull_request:
+    paths:
+      - "operate/client/**"
+      - "!.github/workflows/zeebe-*"
+      - "!dist/**"
+      - "!zeebe/**"
+      - "!spring-boot-starter-camunda-sdk/**"
```

Comment:
This is the same logic as used in [operate-frontend.yml](https://github.com/camunda/zeebe/blob/main/.github/workflows/operate-frontend.yml)